### PR TITLE
CGI unescape S3 key

### PR DESCRIPTION
**Pull request by IvanVergiliev**
From http://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html:

> The s3 key provides information about the bucket and object involved in the event. Note that the object keyname value is URL encoded. For example "red flower.jpg" becomes "red+flower.jpg".

### CODE REVIEW

**Comment by repeatedly** [comment]
@okkez Could you check this?

**Comment by okkez** [comment]
OK, I will check this PR later.

File: lib/fluent/plugin/in_s3.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require 'fluent/log-ext'
 
 require 'aws-sdk-resources'
+require 'cgi'
```
Comment (okkez) [inline lib/fluent/plugin/in_s3.rb]: Use `require 'cgi/util'`, please.
Becasue `require 'cgi'` loads all libraries related to CGI, but we don't use all cgi feature.

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**